src/blas_like/level1/MaxAbs.cpp

```diff
@@ -0,0 +1,56 @@
+/*
+   Copyright (c) 2009-2016, Jack Poulson
+   All rights reserved.
+
+   This file is part of Elemental and is under the BSD 2-Clause License,
+   which can be found in the LICENSE file in the root directory, or at
+   http://opensource.org/licenses/BSD-2-Clause
+*/
+#include <El-lite.hpp>
+#include <El/blas_like/level1.hpp>
+
+namespace El {
+
+    template<typename Real,typename>
```

Comment:
Elemental's style is to not indent after namespace declarations. Often namespaces can be several deep and it becomes untenable to add four spaces after each. Cf. Google's C++ style guide for this decision.